lib/publiccloud/instance.pm

```diff
@@ -372,7 +372,7 @@ Return:
 sub wait_for_ssh {
     my ($self, %args) = @_;
     # Input parameters, see description in above head2 - Parameters section:
-    $args{timeout} //= 600;
+    $args{timeout} //= get_var('PUBLIC_CLOUD_SSH_TIMEOUT', 600);
```

Comment:
you can remove `//` operator as get_var returns default now

Reply:
But it would only use `get_var` if not explicitly defined in `%args`, so I think removing the `//` would alter the intended use case.

Reply:
As we discussed, this  expression was introduced here just to tweak the _default_ value for `debugging purposes`, via test settings, _when the input parameter is empty_, either to force a timeout (just assigning in clone command a proper low value, like PUBLIC_CLOUD_SSH_TIMEOUT=1 or 2,3..) or to force never timeout (a very high value). The `normal` routine flow shall still rely in the last coded default. But indeed do **never apply** it when a `timeout` value is passed in the routine call: it that last case that value shall be used.

Reply:
As explained, I realized that just when the input value is explicitly defined, the debug-tweaking cannot be used anymore.
Therefore I changed that parameter to be prior when set for debuugging, and the `input // constant` when empty, that is the normal code-flow.